e2e/docker-compose.yaml

```diff
@@ -39,7 +39,7 @@ services:
     user: '$UID'
     volumes:
       - .:/e2e
-    command: 'sh -c "npm ci --omit=dev && npm run lint && npx wait-on http://localhost:8080/debug/ready"'
+    command: 'sh -c "npm ci --omit=dev && npm run lint && npx wait-on -v -l http://host.docker.internal:8080/debug/ready"'
```

Comment:
This should fix WSL support for this compose file - without this npx wait-on never finishes for me.